pilot/pkg/serviceregistry/kube/controller/ambient/workloads.go

```diff
@@ -728,12 +730,19 @@ func workloadEntryWorkloadBuilder(
 		w.WorkloadType = workloadapi.WorkloadType_POD // XXX(shashankram): HACK to impersonate pod
 		w.CanonicalName, w.CanonicalRevision = kubelabels.CanonicalService(wle.Labels, w.WorkloadName)
 
+		scopeForService := make(map[string]model.ServiceScope)
+		for _, service := range services {
+			// TODO:  Should the scopo come from the service entry? But the service entry will always be local?
+			scopeForService[service.ResourceName()] = model.Local
```

Comment:
Yeah for workload/serviceenetry, I'm pretty sure the scope of our local instances will always be local